build.assets/windows/build.ps1

```diff
@@ -444,6 +444,44 @@ function Build-Connect {
     Write-Host $("Built Teleport Connect in {0:g}" -f $CommandDuration)
 }
 
+function Write-Version-Objects {
+    [CmdlesBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $TeleportSourceDirectory,
+        [Parameter(Mandatory)]
+        [string] $TeleportVersion,
+    )
+    Write-Host "Generating version info files for tsh.exe and tctl.exe..."
+
+    # install go-winres (v0.3.3)
+    go install github.com/tc-hib/go-winres@d743268d7ea168077ddd443c4240562d4f5e8c3e
```

Comment:
Can we just use `rc.exe` instead? https://learn.microsoft.com/en-us/windows/win32/menurc/resource-compiler

This would remove the dependency, as it _should_ already be available in the build environment.

Reply:
We could, and we do for some other Windows software (though we use `windres`, not `rc.exe`), but we're trying to get away from it because it's difficult to work with.

You have to maintain [C-style code](https://github.com/gravitational/teleport.e/blob/master/windowsauth/version.rc) that is difficult to template, then you have to do some [tricky string manipulation](https://github.com/gravitational/teleport.e/blob/master/windowsauth/Makefile#L25-L31) because the tool is very strict about its input.

The Go equivalent works consistently on Linux and Windows, accepts our existing tag with no need for further string manipulation, and doesn't require us to maintain any other metadata files.

I would much rather maintain a few lines in a script that simply pass the existing tag and metadata into a CLI invocation than the platform-specific spaghetti monster that we've done in the past, but I can see the argument for using the existing tooling if you feel strongly.